### PR TITLE
sys-kernel/coreos-*: bump to 4.4.3

### PR DESCRIPTION
**Pull request by vcaputo**

### CODE REVIEW

File: sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch [hunk 1]
```diff
@@ -1,4 +1,4 @@
-From 64ef0efdd90f5aae4fae7c76783b09af53d29dfe Mon Sep 17 00:00:00 2001
+From 78e4150dc214a67ee53b9d2fc0a3d893286d21eb Mon Sep 17 00:00:00 2001
```
Comment (crawford) [inline sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch]: Why did this patch change?

Comment (marineam) [inline sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch]: Rebased on to the patch formally  known as 0019-overlayfs-use-a-minimal-buffer-in-ovl_copy_xattr.patch I presume.

Comment (vcaputo) [inline sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch]: 0019-overlayfs-use-a-minimal-buffer-in-ovl_copy_xattr.patch is upstream in 4.4.3 (woot)

so we drop 0019 and amend this patch to accommodate the new copy_up.

Comment (vcaputo) [inline sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch]: Note that it doesn't really change, all that's been added is the new context lines.

Comment (crawford) [inline sys-kernel/coreos-sources/files/4.4/0015-Overlayfs-Use-copy-up-security-hooks.patch]: Ohhhhhhhhhhhhhhhh, the changes are the context, not the actual diff content. Diff-n-diff is hard to parse.

**Comment by crawford** [comment]
LGTM

**Comment by marineam** [comment]
LGTM

**Comment by vcaputo** [comment]
hmm, my test build finally reached overlayfs and seems ot have failed, so it appears that patch is going to need more mucking.

DO NOT MERGE yet

File: sys-kernel/coreos-sources/files/4.4/z0015-Overlayfs-Use-copy-up-security-hooks.patch [hunk 2]
```diff
@@ -13,25 +13,25 @@ Signed-off-by: David Howells <dhowells@redhat.com>
...
- 			goto out_free_value;
+@@ -70,6 +70,14 @@ retry:
+ 			value_size = size;
+ 			goto retry;
  		}
 +		error = security_inode_copy_up_xattr(old, new,
 +						     name, value, &size);
 +		if (error < 0)
-+			goto out_free_value;
++			break;
```
Comment (vcaputo) [inline sys-kernel/coreos-sources/files/4.4/z0015-Overlayfs-Use-copy-up-security-hooks.patch]: Upstream dropped the out_free_value label, so this becomes a break.

**Comment by vcaputo** [comment]
Build succeeds here now, that was annoying.

PTAL

**Comment by crawford** [comment]
LGTM
